frontend/src/components/Layout/Menu/MenuContainer.tsx

```diff
@@ -1,33 +1,32 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 
-import { PATHS } from 'constants/routes';
 import { ROLES_MENU } from 'constants/menuRoles';
 import { useGetProfile } from 'api/profile';
-import { setCurrentMenuPath, getCurrentMenuPath } from 'utils/helpers/selectMenuHelpers';
 
 import Menu from './Menu';
 import { useListStyles } from './styled';
 
-const MenuContainer: React.FC = () => {
+interface Props {
+  isSqueeze?: boolean;
+  handleSqueeze?: () => void;
+}
+
+const MenuContainer: React.FC<Props> = ({ handleSqueeze, isSqueeze }) => {
   const { data } = useGetProfile();
   const menuRole = data?.role;
   const menuItems = menuRole ? ROLES_MENU[menuRole] : [];
   const { pathname } = window.location;
-  const getInitMenuName = getCurrentMenuPath();
-  const [menuItem, setMenuItem] = useState<string | undefined>(getInitMenuName);
-
-  useEffect(() => {
-    const PATH_VALUES = Object.values(PATHS);
-    PATH_VALUES.forEach((route) => {
-      if (pathname.includes(route) && route !== PATHS.home) {
-        setMenuItem(route);
-        setCurrentMenuPath(route);
-      }
-    });
-  }, [pathname]);
   const classes = useListStyles();
 
-  return <Menu menuList={menuItems} menuItem={menuItem} classes={classes} />;
+  return (
+    <Menu
+      menuList={menuItems}
+      classes={classes}
+      pathname={pathname}
+      isSqueeze={isSqueeze}
```

Comment:
Error in console https://prnt.sc/26olfek

Reply:
Fixed and rewrote